test/plugins/IframeGeometryIdPlugin.test.js

```diff
@@ -53,6 +53,26 @@ describe('IframeGeometryIdPlugin', () => {
 		expect(iframeSpy).toHaveBeenCalledWith(IFRAME_GEOMETRY_REFERENCE_ID, expectedGeometryId);
 	});
 
+	it('updates the iframes data attribute when drawing is deleted', async () => {
```

Comment:
Let's name it `registers draw.fileSaveResult listeners and updates the iframes data attribute`
and delete lines 42-54